contracts/infrastructure/dapp/YearnV2Filter.sol

```diff
@@ -0,0 +1,64 @@
+// Copyright (C) 2021  Argent Labs Ltd. <https://argent.xyz>
+
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+// SPDX-License-Identifier: GPL-3.0-only
+pragma solidity ^0.8.3;
+
+import "./BaseFilter.sol";
+
+/**
+ * @title YearnV2Filter
+ * @notice Filter used for deposits & withdrawals into/from YearnV2
+ * @author Olivier VDB - <olivier@argent.xyz>
+ */
+contract YearnV2Filter is BaseFilter {
```

Comment:
Can you add the deployment address of one (or more) vaults in the description of the filter.